### DISCUSSION TITLE
Sentry sending mails via Postal

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#2086] [Help with using Postal]
Anyone here using postal to send mails from self hosted sentry?
https://develop.sentry.dev/services/email/#outbound-email

If so, please provide details what settings you have in use in sentry for mail.

Right now we use gmail, yet would like to migrate to postal, yet never was able to make it work in past,

<img width="636" alt="image" src="https://user-images.githubusercontent.com/6324047/181586354-448e32b3-7f30-4083-8e48-77bc72c6c505.png">

**Comment by willpower232** [comment]
What problems did you have? It shouldn't be much different to integrating any other service
